src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/LayoutViewGroup.Android.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using Android.Content;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Rectangle = Xamarin.Forms.Rectangle;
+using Size = Xamarin.Forms.Size;
+
+namespace Xamarin.Platform.Handlers
+{
+	public class LayoutViewGroup : ViewGroup
+	{
+		public LayoutViewGroup(Context context) : base(context)
+		{
+		}
+
+		public LayoutViewGroup(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
+		{
+		}
+
+		public LayoutViewGroup(Context context, IAttributeSet attrs) : base(context, attrs)
+		{
+		}
+
+		public LayoutViewGroup(Context context, IAttributeSet attrs, int defStyleAttr) : base(context, attrs, defStyleAttr)
+		{
+		}
+
+		public LayoutViewGroup(Context context, IAttributeSet attrs, int defStyleAttr, int defStyleRes) : base(context, attrs, defStyleAttr, defStyleRes)
+		{
+		}
+
+		protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
+		{
+			if (Context == null)
+			{
+				return;
+			}
+
+			if (CrossPlatformMeasure == null)
+			{
+				base.OnMeasure(widthMeasureSpec, heightMeasureSpec);
+				return;
+			}
+
+			var deviceIndependentWidth = widthMeasureSpec.ToDouble(Context);
+			var deviceIndependentHeight = heightMeasureSpec.ToDouble(Context);
+
+			var size = CrossPlatformMeasure(deviceIndependentWidth, deviceIndependentHeight);
+
+			var nativeWidth = Context.ToPixels(size.Width);
+			var nativeHeight = Context.ToPixels(size.Height);
+
+			SetMeasuredDimension((int)nativeWidth, (int)nativeHeight);
+		}
+
+		protected override void OnLayout(bool changed, int l, int t, int r, int b)
+		{
+			if (CrossPlatformArrange == null || Context == null)
+			{
+				return;
+			}
+
+			var deviceIndependentLeft = Context.FromPixels(l);
+			var deviceIndependentTop = Context.FromPixels(t);
+			var deviceIndependentRight = Context.FromPixels(r);
+			var deviceIndependentBottom = Context.FromPixels(b);
+
+			var destination = Rectangle.FromLTRB(deviceIndependentLeft, deviceIndependentTop,
```

Comment:
Account for RTL? #3066 🤔

Reply:
When we implement margins, we will need to account for it.